assets/bem/blocks/_book.scss

```diff
@@ -0,0 +1,58 @@
+.book {
+    &--120 {
```

Comment:
do all books have this width? If so why do we need the modifier class?

Reply:
I think that books might have different width on different pages and blocks in future.